src/services/shopify-client.doubles.ts

```diff
@@ -15,12 +10,12 @@ const fakeData = [
       id: 1251,
       amountSubscribers: 2,
       state: SubscriberListState.ready,
-    }
-  }
+    },
+  },
 ];
 
 export class HardcodedShopifyClient implements ShopifyClient {
-  public async getShopifyData(email: string, apiKey: string): Promise<ConnectedShops[]> {
+  public async getShopifyData(): Promise<ConnectedShops[]> {
```

Comment:
In my opinion, the result type should be a wrapper that takes into account errors.

Reply:
Yes you are right, I will make a small refactor to reuse dopplerLegacyClient results. In a separate commit.